SlicerCART/src/SlicerCART.py

```diff
@@ -31,7 +36,8 @@
     "PyYAML": "yaml",
     "pynrrd": "nrrd",
     "slicerio": "slicerio",
-    "bids_validator": "bids_validator"
+    "bids_validator": "bids_validator",
```

Comment:
On top of this block, you should add a comment explaining why you define those variable on top of the initial method.